File: 251011/약수가 세 개인 수/numbers-with-three-divisors.cpp
#include <iostream>

using namespace std;

int st, ed;

int main() {
    cin >> st >> ed;

    // Please write your code here.
    //int sum = 0;
    int num = 0;

    for(int i = st; i <= ed; i++) {
        int sum = 0;
        for(int j = 1; j <= i; j++) {
            if(i % j == 0) {
                sum++;
            }
        }
        if(sum == 3) {
            num++;
        }
    }
    cout << num;
    return 0;
}

File: 251114/약수가 세 개인 수/numbers-with-three-divisors.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        // Please write your code here.
        var input = Console.ReadLine().Split();
        int a = int.Parse(input[0]);
        int b = int.Parse(input[1]);
        int count = 0;

        for(int i = a; i <= b; i++) {
            int count_div = 0;

            for(int j = 1; j <= i; j++) {
                if(i % j == 0) {
                    count_div++;
                }
            }
            if(count_div == 3) {
                count++;
            }
        }
        Console.Write(count);
    }
}
